From 73ef9126329f13a8d377bcb8f2c6d6d7cb500e42 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:34:42 -0400
Subject: [alsa] hda/realtek: New vendor ID for ALC233

Message-id: <140972823573-329-patchset-jkysela@redhat.com>
Patchwork-id: 91025
O-Subject: [RHEL 7.1 PATCH 328/442] ALSA: hda/realtek - New vendor ID for ALC233
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

This is compatible with ALC255.
It is use for Lenovo.

Signed-off-by: Kailang Yang <kailang@realtek.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 92f974df346013160481324c39851e7ea9ce482a)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index 2d698c0..e097e80 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -6225,6 +6225,7 @@ static const struct hda_codec_preset snd_hda_preset_realtek[] = {
  { .id = 0x10ec0221, .name = "ALC221", .patch = patch_alc269 },
  { .id = 0x10ec0231, .name = "ALC231", .patch = patch_alc269 },
  { .id = 0x10ec0233, .name = "ALC233", .patch = patch_alc269 },
+ { .id = 0x10ec0235, .name = "ALC233", .patch = patch_alc269 },
  { .id = 0x10ec0255, .name = "ALC255", .patch = patch_alc269 },
  { .id = 0x10ec0260, .name = "ALC260", .patch = patch_alc260 },
  { .id = 0x10ec0262, .name = "ALC262", .patch = patch_alc262 },
-- 
1.7.1